xc/common/cmake/arch_define.cmake

```diff
@@ -61,11 +61,24 @@ function(ADD_XC_ARCH_DEFINE)
   set(YOSYS_CONV_SCRIPT ${ADD_XC_ARCH_DEFINE_YOSYS_CONV_SCRIPT})
   set(YOSYS_UTILS_SCRIPT ${ADD_XC_ARCH_DEFINE_YOSYS_UTILS_SCRIPT})
   set(YOSYS_TECHMAP ${symbiflow-arch-defs_SOURCE_DIR}/xc/${FAMILY}/techmap)
+
+  # Notes on optimized flag settings:
+  # These flags have been optimized for the ibex and baselitex designs.
+  # - acc_fac: Lowering this to 0.7 slightly improves runtime 1-4% (default: 1)
+  # - astar_fac: Increasing this to 1.8 reduces runtime 2-15% (default: 1.2)
+  # - initial_pres_fac: Setting this to 2.828 reduces runtime 10-20% from the default,
+  #   and up to 3% over the previously used value of 4 (default: 0.5)
+  # - place_delay_model: `delta` is up to 5% faster than `delta_override` with slight to no
+  #   degradation in QoR (default: delta)
+  # - pres_fac_mult: A slightly lower value of 1.2 performs better on average by ~5% for some designs (default: 1.3)
+  # Based on analysis performed on hydra.vtr.tools for the ibex, baselitex, and bram-n3 designs.
+  # These changes did not have a measurable effect on QoR for these designs.
+  # More details can be found in the report: https://colab.research.google.com/drive/1X91RGZnvlC7dBjJJUbS7JfqCbPCzJ3Xb
```

Comment:
Thanks for the additional comments.  My confusion at this point is how from the data you posted did you select these particular settings for `acc_fac`/`astar_fac`/`initial_pres_fac`/`pres_fac_mult`.

Reply:
I started with parameters that resulted in the shortest runtime for each project, and I keep the parameters that agree. Then, for the remaining parameters, I try values and check that they rank highly among all projects. I use the statistics to check that the selected parameters rank well with mean and minimum values, and that fmax doesn't suffer. I also look at the scatter plot to get an idea of the larger population of configurations, because I want to avoid parameter values that I don't think will generalize.

It's not very scientific, and I'm not sure how to automate it yet. Maybe ML :)

Reply:
I'm not interested in full automation here, as I agree that is not easy.  This comment was not intended to get a reply here, but to modify the comment and/or Colab to capture the process.

Are there some short hands you can add to the Colab for the process of boiling it down?  Writing down what you did in a process is probably the minimum here, but ideally there'd be a way for others to follow what you've done.

Reply:
I've expanded on this a bit in the [Colab](https://colab.research.google.com/drive/1X91RGZnvlC7dBjJJUbS7JfqCbPCzJ3Xb?usp=sharing).

Reply:
Thanks. I realized this isn't that bad to automate -- just sort by the minimum percent improvement across some representative projects. I've added this table to the colab, and found some better parameters that give a 15% runtime improvement on both `baselitex` and `ibex`. Thanks for the discussion!

Reply:
>  I've added this table to the colab, and found some better parameters that give at least 15% runtime improvement on both baselitex and ibex

That's good news! Given that the Colab page now really has code in it to select parameters, we should probably manage it like code, e.g. check it in.  Can you export the current Colab page (e.g. the `.ipynb`) and add it to this PR?

Reply:
I like the "sort by improvement" approach.  Should that table also have relative CPD effect column for quickly evaluating the potential cost to QoR against the benefit to runtime?  I think with that, it is straight forward enough to pick a point with reasonable improvement with minimal to no QoR cost.

Reply:
I added a "cpd_improvement" column. The parameters I chose increased CPD by 5%. I chose not to modify `--first_iter_pres_fac` because it made `xc/xc7/tests/counter/counter_basys3` unroutable .

Reply:
I've checked in the colab at `utils/ipynb/Parameter_Sweep_using_fpga_tool_perf.ipynb`.